test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -29,6 +30,19 @@ TEST(QuicPlatformTest, QuicArraysize) {
   EXPECT_EQ(5, QUIC_ARRAYSIZE(array));
 }
 
+enum class TestEnum { ZERO = 0, ONE, TWO, COUNT };
+
+TEST(QuicPlatformTest, QuicClientStats) {
+  // Just make sure they compile.
```

Comment:
On functions where we're not implementing the actual functionality, should we have comments or a document somewhere for what expect to work and what not?

Reply:
We can 1) add comments to quic_client_stats_impl.h, 2) add a column to the [platform impl tracking spreadsheet](https://docs.google.com/spreadsheets/d/1vcd8ERD1CRSECu2gU-Xmpd6Vj0GpC8mT_-BvhU7b6EM/edit#gid=0) to say it's not actually implemented.

Or both?  WDYT?

Reply:
I'd mildly prefer something in the code just so folks don't have to dig up the spreadsheet but as long as we're tracking it I'm good :-)

Reply:
Added a comment in quic_client_stats_impl.h.